pyproject.toml

```diff
@@ -0,0 +1,126 @@
+[build-system]
+requires = ["setuptools>=64", "wheel", "setuptools_scm[toml]>=3.4.3"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "oculy"
+dynamic=["version"]
+description = "Visualisation app for Enaml based plugin applications"
+readme = "README.rst"
+requires-python = ">=3.7"
+license = {file = "LICENSE"}
+authors = [
+  {name = "Matthieu C. Dartiailh", email = "m.dartiailh@gmail.com"}
+]
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "License :: OSI Approved :: BSD License",
+    "Natural Language :: English",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+]
+
+[options]
+packages = "find:"
+zip_safe = false
+long_description = "file: README.rst"
+long_description_content_type = "text/x-rst"
+platforms = "Linux; Windows; Mac"
```

Comment:
This is either cover by the default config of setuptools or repeat what is listed above so please remove it.

Reply:
I agree with you on **long_description** and **plateforms**.
What about *long_description_content_type = "text/x-rst"* and *zip_safe*? Are they covered in *project" table?

Reply:
The long description formqt does not make any sense without the long description. For zip_safe I am nor sure anymore what it does with modern build system. So we can remove it and we will see later if it causes any issue (I removed it from glaze and for the basic stuff we are fine).

Reply:
Typically python packages are installed as zip files, meaning that their source is not available as files on the file system. The `zip_safe = True` flag means that this is okay for your package. If you make use of `__file__` attributes to find included data files in your package you will probably need to set `zip_safe = False`.